cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -369,6 +377,17 @@ protected Labels getLabelsWithStrimziName(String name, Map<String, String> addit
         return labels.withStrimziName(name).withAdditionalLabels(additionalLabels);
     }
 
+    /**
+     * @param name the value for the {@code strimzi.io/name} key
+     * @param podName the value for the {@code strimzi.io/pod-name} key
+     * @param additionalLabels a nullable map of additional labels to be added to this instance of Labels
+     *
+     * @return Labels object with the default labels merged with the provided additional labels and the new {@code strimzi.io/name} label
+     */
+    protected Labels getLabelsWithStrimziName(String name, String podName, Map<String, String> additionalLabels) {
```

Comment:
isn't it better having just `getLabelsWithStrimzi`? The `Name` part seems to imply that you look for only `strimzi.io/name` label (and you are using `withStrimziName` inside) not for `PodName` as well (`strimzi.io/pod-name` label).

Reply:
I renamed it to `getLabelsWithStrimziNameAndPodName` -> I think that is in line with `getLabelsWithStrimziNameAndDiscovery` and other methods. The Pod name in the label is significant as it is needed for the StrimziPodSet pods only.